gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineVersionsIntegrationTest.groovy

```diff
@@ -174,21 +175,42 @@ class BaselineVersionsIntegrationTest  extends AbstractPluginTest {
         result.output.contains("  bom:\n" +
                 "                  org.scala-lang:scala-compiler -> 2.12.5\n" +
                 "                  org.scala-lang:scala-library -> 2.12.5")
-
     }
 
     def 'Version props conflict should succeed'() {
         when:
-        setupVersionsProps("com.google.guava:* = 22.0\ncom.google.guava:guava = 23.0")
-        buildFile << standardBuildFile(projectDir) + """
+        setupVersionsProps("""
+            com.fasterxml.jackson.*:* = 2.9.3
+            com.fasterxml.jackson.core:jackson-annotations = 2.9.5
+        """.stripIndent())
+        buildFile << standardBuildFile(projectDir)
+        buildFile << """
         dependencies {
-            compile 'com.google.guava:guava'
```

Comment:
original test was failing with the newly corrected logic, because `com.google.guava:*` is not being used by any artifact.